src/pages/tx/[tid].tsx

```diff
@@ -1,1114 +1,261 @@
 import BigNumber from "bignumber.js";
 import clsx from "clsx";
-import { Dispatch, SetStateAction, useState } from "react";
-import { FiCopy, FiChevronUp, FiChevronDown } from "react-icons/fi";
+import { FiX } from "react-icons/fi";
+import { IoMdCheckmarkCircle } from "react-icons/io";
 
-import { useUnitSuffix } from "hooks/useUnitSuffix";
-import useWindowDimensions from "hooks/useWindowDimensions";
 import { formatDateToUTC, getDuration } from "shared/durationHelper";
 import { transformTransactionData } from "shared/transactionDataHelper";
 import { truncateTextFromMiddle } from "shared/textHelper";
 import { GWEI_SYMBOL } from "shared/constants";
 import TransactionsApi from "@api/TransactionsApi";
-import { TransactionI, TransactionType } from "@api/types";
+import { TransactionI } from "@api/types";
 
 import { SearchBar } from "layouts/components/searchbar/SearchBar";
 import GradientCardContainer from "@components/commons/GradientCardContainer";
 import NumericFormat from "@components/commons/NumericFormat";
 import LinkText from "@components/commons/LinkText";
-import Tooltip from "@components/commons/Tooltip";
-import { InfoIcon } from "@components/icons/InfoIcon";
-import { ConfirmCheck } from "@components/icons/ConfirmCheck";
-import { RejectedCross } from "@components/icons/RejectedCross";
-import { GreenTickIcon } from "@components/icons/GreenTickIcon";
 import { NetworkConnection } from "@contexts/Environment";
+
 import BoldedTitle from "./_components/BoldedTitle";
+import GasDetails from "./_components/GasDetails";
+import RawInput from "./_components/RawInput";
+import WithCopy from "./_components/WithCopy";
 
 function Transaction({ txDetails }: { txDetails: TransactionI }) {
-  const amount = { value: txDetails.value, symbol: txDetails.symbol };
-  const transactionFee = { value: txDetails.fee, symbol: txDetails.symbol };
   const gasPrice = { value: txDetails.gasPrice, symbol: GWEI_SYMBOL };
   const gasUsedPercentage = new BigNumber(txDetails.gasUsed)
     .dividedBy(txDetails.gasLimit)
     .multipliedBy(100)
     .toFixed(2);
+  const timeDuration = getDuration(Number(txDetails.timeInSec));
+  const timeInUTC = formatDateToUTC(txDetails.timestamp);
+
+  const titleFontCss =
+    "text-white-700 break-all leading-[22.4px] tracking-[0.01em] lg:leading-[24px] lg:-tracking-[0.02em]";
+  const valueFontCss = "text-white-50 leading-[24px] -tracking-[0.02em]";
+  const rowCss =
+    "flex md:flex-col justify-between md:justify-normal gap-y-2 h-6 md:h-auto";
 
   return (
-    <div className="px-1 md:px-0 mt-12">
+    <div className="px-1 md:px-0 mt-12 antialiased">
```

Comment:
<img width="249" alt="image" src="https://github.com/WavesHQ/metascan/assets/13292662/26bf711e-c093-42fb-b67c-34deb91d3ea3">
Very responsive! one minor thing, can add minimum width to handle this

Reply:
Updated! Thanks for catching!